shell/key-bindings.fish

```diff
@@ -1,7 +1,9 @@
 # Key bindings
 # ------------
 function fzf_key_bindings
-
+  set -g __FZF_FISH_MAJOR (echo $FISH_VERSION | cut -f1 -d.)
```

Comment:
This creates a global variable, which isn't ideal. Ideally we'd be able to set a variable that fzf-history-widget has access to, but so far I've been unable to find a way.